### PR TITLE
OSG 3.4 is now the default (SOFTWARE-2622)

### PR DESCRIPTION
**Pull request by edquist**

### CODE REVIEW

File: osg-wn-nightly-build [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-    3.3-devel-el7
-    3.3-testing
-    3.3-testing-el6
-    3.3-testing-el7
+    3.4-devel
+    3.4-devel-el6
+    3.4-devel-el7
+    3.4-testing
+    3.4-testing-el6
+    3.4-testing-el7
```
Comment (brianhlin) [inline osg-wn-nightly-build]: I think we want to add 3.4 branches but not remove the 3.3 branches. Same applies for `update-all`

**Comment by edquist** [comment]
@bbockelm, @brianhlin says maybe you can give this a review (@matyasselmeci is out).

I am not sure if this is correct, or if (say) the 3.4 branches need to be added to the 3.3 list instead of replacing the 3.3 ones...

File: osg-wn-nightly-build (hunk 1 above)
Comment (edquist) [inline osg-wn-nightly-build]: How now brown cow?

Comment (brianhlin) [inline osg-wn-nightly-build]: That certainly looks better to me but I'm not sure if anything else needs to be touched.

**Comment by brianhlin** [comment]
I had a chance to look at this more closely and the changes LGTM.

Now the question is where does the nightly builder run?

**Comment by bbockelm** [comment]
Would it be better to do this across two PRs - one to verify 3.4 dev / testing works, then another for production?

**Comment by edquist** [comment]
Sure. Updated this PR to only include devel & testing; once that goes in we can make a separate one to add in `3.4`, `3.4-el6`, `3.4-el7` to `branches` in `update-all`.

**Comment by matyasselmeci** [comment]
These branches don't actually exist on opensciencegrid/docker-osg-wn yet...

**Comment by brianhlin** [comment]
Can we get something in the `README` describing the following?

# How to add new tags
# Where the automated nightly builds run and how to update them when adding new tags

**Comment by matyasselmeci** [comment]
@edquist, could you split your `genbranches` changes into a separate PR? We need to create those branches before having the update scripts run on them.

**Comment by edquist** [comment]
@matyasselmeci, do you mean just adding 3.4 to `SERIESES`, or updating `DEFAULT_SERIES` too?

**Comment by matyasselmeci** [comment]
Yeah, both.

**Comment by edquist** [comment]
Ok, I've updated this PR to include just the `genbranches` changes, and the rest are now in #2

**Pull request merged**